types/src/transaction/analyzed_transaction.rs

```diff
@@ -131,11 +133,18 @@ impl AnalyzedTransaction {
             // receiver account resource.
             write_hints.push(Self::account_resource_location(receiver_address));
         }
+
+        let read_hints = vec![
+            Self::current_ts_location(),
+            Self::features_location(),
+            Self::aptos_coin_info_location(),
+            Self::chain_id_location(),
+            Self::transaction_fee_burn_cap_location(),
```

Comment:
Also, how about account creation txn?

Reply:
Will need to print some logs to figure out and confirm. I'd test the performance impact with p2p first.

Reply:
fails your test already :trollface: 
test_no_conflict_across_shards_in_non_last_rounds

Reply:
Let's rebase once https://github.com/aptos-labs/aptos-core/pull/9883 lands. I moved all these tests from v1 to v2. We will see if they fail in V2. cc - @zjma